exporters/dsexporter/dsexporter_test.go

```diff
@@ -16,31 +18,48 @@ func TestCustomCollector(t *testing.T) {
 	metricFamilies, err := metrics.Gather()
 	assert.NoError(t, err)
 
-	var RequestCountValue float64
+	var konfluxProbeSuccessValue float64
 	for _, mf := range metricFamilies {
-		if mf.GetName() == "request_count" {
-			RequestCountValue = mf.GetMetric()[0].GetCounter().GetValue()
+		if mf.GetName() == "konflux_probe_success" {
+			konfluxProbeSuccessValue = mf.GetMetric()[0].GetCounter().GetValue()
 			break
 		}
 	}
 
 	// Check whether the exported metric value is initially 0.
-	assert.Equal(t, float64(0), RequestCountValue)
+	assert.Equal(t, float64(0), konfluxProbeSuccessValue)
 
-	// Increment the requestCounter by calling the Inc method.
-	exporter.requestCounter.Inc()
+	// Increment the konfluxProbeSuccess by calling the Inc method.
+	exporter.konfluxProbeSuccess.With(prometheus.Labels{"service":"test_service", "check":"github"}).Inc()
 
 	// Collecting metrics again
 	metricFamilies, err = metrics.Gather()
 	assert.NoError(t, err)
 
 	for _, mf := range metricFamilies {
-		if mf.GetName() == "request_count" {
-			RequestCountValue = mf.GetMetric()[0].GetCounter().GetValue()
+		if mf.GetName() == "konflux_probe_success" {
+			konfluxProbeSuccessValue = mf.GetMetric()[0].GetCounter().GetValue()
 			break
 		}
 	}
 
 	// Check whether the exported metric value is now 1 after incrementing.
-	assert.Equal(t, float64(1), RequestCountValue)
+	assert.Equal(t, float64(1), konfluxProbeSuccessValue)
+}
+
+func TestcheckIfServiceUp(t *testing.T) {
```

Comment:
Need to also test the scenario in which the service is not up.

Reply:
Updated